pandora/client.py

```diff
@@ -263,6 +262,10 @@ def get_ad_item(self, station_id, ad_token):
         from .models.pandora import AdItem
 
         ad_metadata = self.get_ad_metadata(ad_token)
+        if not station_id:
```

Comment:
Can you validate this before the call to `get_ad_metadata`? There's no point in making an expensive network call if we don't have enough parameters to handle the response.